src/Components/Nav/index.js

```diff
@@ -8,32 +9,66 @@ export default class Nav extends Component {
     super(props);
     this.state = {
       isSearchbarOn: false,
+      isOpenAside: false,
     };
   }
 
-  searchbarOn = () => {
+  onToggleSearch = () => {
+    const { isSearchbarOn } = this.state;
     this.setState({
-      isSearchbarOn: true,
+      isSearchbarOn: !isSearchbarOn,
     });
   };
 
-  searchbarOff = () => {
-    this.setState({
-      isSearchbarOn: false,
-    });
+  // searchbarOn = () => {
+  //   this.setState({
+  //     isSearchbarOn: true,
+  //   });
+  // };
+
+  // searchbarOff = () => {
+  //   this.setState({
+  //     isSearchbarOn: false,
+  //   });
+  // };
+
+  onToggleSideMenu = (e) => {
+    const { isOpenAside } = this.state;
+    const classList = [...e.target.classList];
+
+    // if (!isOpenAside) {
+    //   this.setState({ isOpenAside: !isOpenAside });
+    // } else {
+    //   if (classList.includes('sideMenuWrap')) {
+    //     this.setState({ isOpenAside: !isOpenAside });
+    //   }
+    // }
+
+    // eslint-disable-next-line no-unused-expressions
+    !isOpenAside
+      ? this.setState({ isOpenAside: !isOpenAside })
+      : classList.includes('sideMenuWrap')
+      ? this.setState({ isOpenAside: !isOpenAside })
+      : null;
```

Comment:
```suggestion
    if (!isOpenAside || classList.includes('sideMenuWrap')) {
      this.setState({ isOpenAside: !isOpenAside })
    }
```
- 이렇게 작성하면 동일한 코드일 것 같은데 같은 동작을 nesting ternary operator로 묶어주신 이유가 있을까요?
- 이전 리뷰에 if문과 ternary operator 간 차이에 대해 언급한 내용이 있으니 참고해주세요! ([링크](https://github.com/wecode-bootcamp-korea/21-1st-KaKa0Adult-frontend/pull/11#discussion_r651496620))